cmd/virt-v2v-monitor/virt-v2v-monitor.go

```diff
@@ -95,37 +90,37 @@ func main() {
 		os.Stdout.Write([]byte("\n"))
 		err := scanner.Err()
 		if err != nil {
-			klog.Fatal("Output monitoring failed! ", err)
+			fmt.Println("virt-v2v monitoring: Output monitoring failed! ", err)
 		}
 
 		if match := COPY_DISK_RE.FindSubmatch(line); match != nil {
 			diskNumber, _ = strconv.ParseUint(string(match[1]), 10, 0)
 			disks, _ = strconv.ParseUint(string(match[2]), 10, 0)
-			klog.Infof("Copying disk %d out of %d", diskNumber, disks)
+			fmt.Printf("virt-v2v monitoring: Copying disk %d out of %d\n", diskNumber, disks)
 			progress = 0
 			err = updateProgress(progressCounter, diskNumber, progress)
 		} else if match := DISK_PROGRESS_RE.FindSubmatch(line); match != nil {
 			progress, _ = strconv.ParseUint(string(match[1]), 10, 0)
-			klog.Infof("Progress update, completed %d %%", progress)
+			fmt.Printf("virt-v2v monitoring: Progress update, completed %d %%\n", progress)
 			err = updateProgress(progressCounter, diskNumber, progress)
 		} else if match := FINISHED_RE.Find(line); match != nil {
 			// Make sure we flag conversion as finished. This is
 			// just in case we miss the last progress update for some reason.
-			klog.Infof("Finished")
+			fmt.Println("virt-v2v monitoring: Finished")
 			for disk := uint64(0); disk < disks; disk++ {
 				err = updateProgress(progressCounter, disk, 100)
 			}
 		} else {
-			klog.V(1).Info("Ignoring line: ", string(line))
+			fmt.Println("virt-v2v monitoring: Ignoring line: ", string(line))
 		}
 		if err != nil {
 			// Don't make processing errors fatal.
-			klog.Error("Error updating progress: ", err)
+			fmt.Println("virt-v2v monitoring: Error updating progress: ", err)
 			err = nil
 		}
 	}
 	err := scanner.Err()
 	if err != nil {
-		klog.Fatal("Output monitoring failed! ", err)
+		fmt.Println("virt-v2v monitoring: Output monitoring failed! ", err)
```

Comment:
same comment as above for replacing klog.Fatal with a simple print

Reply:
same.